### ISSUE TITLE
PD/SSFA - changes to Amendments

### ISSUE THREAD

**Issue by acory** [unicef/etools-issues#467]
The Amendment Type field will now be a multiselect drop-down with the following options: 
Dates
Results
Budget
Other (If other is selected, then a comment box is available to provide further details)

**Comment by acory** [comment]
@evrimm @robertavram 
When status is Active or Signed , PD/SSFA can be modified only through an Amendment . 
Is this case is the Review-and-Sign tab readonly also? (Except Fr number when status=Signed)
Can anything be edited through an amendment in the Review-and-Sign tab?

**Comment by evrimm** [comment]
@acory Let's ask @mpawlowski85

**Comment by mpawlowski85** [comment]
@acory You Are correct - for amendments, only the details tab will be open for editing. 
The FR Number field is always open for editing in status=Signed, even if there is no amendment added. 
M.

**Comment by roxanavlad** [comment]
@acory 
test result: pass
tested on demo environment on Chrome and IE

**Comment by janpanchalingam1** [comment]
Passed testing
